### DISCUSSION TITLE
3kh0.github.io links

### DISCUSSION THREAD

**Discussion by fnfgo** [3kh0/ext-remover#393] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
So i need some links for 3kh0 site but since my school is such a bitch they blocked the github repo so i cant fork it or anything.
SOMEONE HELP

**Comment by 3kh0** [comment, marked as answer]
Use a proxy

**Comment by rwg100** [comment]
[https://3kh0githubio-replit-1.rainwang2.repl.co](url)

its 3kho.github.io

**Reply by fnfgo** [reply]
> [https://3kh0githubio-replit-1.rainwang2.repl.co](url)
> 
> its 3kho.github.io

3kh0 2kh0 9kh0 7kh0 and 6kh0 are all blocked
my school sucks

**Reply by Aka-but-nice** [reply]
No, it's not your school blocking each individual link, your school's blocker automatically blocks all sites with the .io at the end, which in this case is 2kh0.github.**io** 3kh0.github.**io**, etc.

**Reply by fnfgo** [reply]
ohh
that makes sense
